docs/src/apireference.md

```diff
@@ -192,8 +191,8 @@ delete(::ModelLike, ::Index)
 [`add_variables`](@ref) while *constrained variables* are
 the variables created with [`add_constrained_variable`](@ref) or
 [`add_constrained_variables`](@ref). Adding constrained variables instead of
-constraining free variables with [`add_constraint`](@ref) allows Variable
-bridges to be used.
+constraining free variables with [`add_constraint`](@ref) allows
+[Variable bridges](@ref) to be used.
```

Comment:
If you capitalize variable here it probably would make sense to capitalize bridges too. I'm not sure I would be capitalizing either though.

Reply:
Ok perhaps this is needed for the ref to work? In that case I'd probably have both capitalized, but if that would be inconsistent with the rest of the repo and it's too much work to change everywhere then don't worry.

Reply:
Fixed